lib/model/bloc/home/HomeViewModelProvider.dart

```diff
@@ -1,19 +1,24 @@
 import 'dart:async';
 
+import 'package:farmsmart_flutter/flavors/app_config.dart';
 import 'package:farmsmart_flutter/model/model/ProfileEntity.dart';
 import 'package:farmsmart_flutter/model/model/loading_status.dart';
 import 'package:farmsmart_flutter/model/repositories/account/AccountRepositoryInterface.dart';
 import 'package:farmsmart_flutter/model/repositories/profile/ProfileRepositoryInterface.dart';
 import 'package:farmsmart_flutter/ui/common/LoadableViewModel.dart';
 import 'package:farmsmart_flutter/ui/common/RefreshableViewModel.dart';
+import 'package:flutter/material.dart';
 
 import '../ViewModelProvider.dart';
 
 class HomeViewModel implements LoadableViewModel, RefreshableViewModel {
   final LoadingStatus loadingStatus;
   final Function refresh;
   final ProfileRepositoryInterface currentProfile;
-  HomeViewModel(this.loadingStatus, this.refresh, this.currentProfile);
+  final bool isDebugBuild;
```

Comment:
I would ask the specific question "debugMenuVisible", then you know exactly what this bool controls.